src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        int MAX_SIZE = 100;
```

Comment:
Hi! I think you could consider making this a final variable!

Reply:
got it thanks!